SwitchItem/tests/SwitchItem-specs.js

```diff
@@ -1,51 +1,46 @@
-import {mount} from 'enzyme';
-import SwitchItem from '../SwitchItem';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
-describe('SwitchItem Specs', () => {
+import SwitchItem, {SwitchItemBase} from '../SwitchItem';
 
+describe('SwitchItem Specs', () => {
 	test('should contain a Switch', () => {
+		render(<SwitchItemBase />);
 
-		const switchItem = mount(
-			<SwitchItem>
-				SwitchItem
-			</SwitchItem>
-		);
+		const expected = 'switch';
+		const actual = screen.getByRole('checkbox').children[2].children[0];
 
-		const expected = 1;
-		const actual = switchItem.find('Switch').length;
-
-		expect(actual).toBe(expected);
+		expect(actual).toHaveClass(expected);
 	});
 
-	test('should pass selected to Switch element', () => {
```

Comment:
is there any reason why this test was deleted?

Reply:
No, just by mistake.